src/arguments.js

```diff
@@ -89,11 +89,11 @@ async function getEventArguments(event) {
         event['width'] = DEFAULT_WIDTH;
     if (event.height === undefined)
         event['height'] = DEFAULT_MIN_HEIGHT;
-    if (event.filename === undefined)
+    if (event.filename === undefined && process.env[ENV_VAR.FILENAME] === undefined)
         event['filename'] = DEFAULT_FILENAME;
-    if (event.subject === undefined)
+    if (event.subject === undefined && process.env[ENV_VAR.EMAIL_SUBJECT] === undefined)
         event['subject'] = DEFAULT_EMAIL_SUBJECT;
-    if (event.note === undefined)
+    if (event.note === undefined && process.env[ENV_VAR.EMAIL_NOTE] === undefined)
```

Comment:
Agreed. will need check how to set default options object though because currently for cli options object is created with commander program, even setting env var is happening at that time for cli https://github.com/opensearch-project/reporting-cli/blob/main/src/arguments.js#L24 and want to keep main logic same for cli/lambda/package. getOptions is setting environment variable only for lambda & credentials as it has `:`. Anyway will add this also to refactor issue and try to prioritize.